draft-private-access-tokens.md

```diff
@@ -226,55 +227,55 @@ The Client sends the Issuer Name to the Mediator so the Mediator know where to
 to the Issuer, so that Mediators cannot learn the ORIGIN_NAME value. Each
-Issuer Name Key is used across all requests on the Issuer, for different Origins.
+ISSUER_NAME_KEY is used across all requests on the Issuer, for different Origins.
 
 ORIGIN_NAME:
-: The Origin name identifies the Origin that requests and verifies Private Access Tokens.
+: The name of the Origin that requests and verifies Private Access Tokens.
 
 ANON_ORIGIN_ID:
-: The Anonymous Origin Identifier is generated by the Client and marked on requests
-to the Mediator and through to the Issuer.
-
-CLIENT_ID:
-: The Client Identifier represents a single client that has authenticated to the Mediator.
-The specifics of this identity are up to the Mediator, but it may be based on an
-attested device identifier or an account login that the Mediator can verify.
-
-ANON_CLIENT_ID:
-: The Anonymous Client Identifier is generated by the Mediator and used when forwarding
-requests to the Issuer. The Mediator maintains a mapping such that there is exactly one
-ANON_CLIENT_ID for each CLIENT_ID + ANON_ORIGIN_ID pair during a specific ISSUER_POLICY_WINDOW.
-
-ANON_ORIGIN_ID_PRIME:
-: The Anonymous Origin Identifier Prime is generated by the Mediator and sent to the Issuer,
-and corresponds to a unique ANON_ORIGIN_ID and a unique ANON_CLIENT_ID.
+: An identifier that is generated by the Client and marked on requests to the
+Mediator, which represents a specific Origin anonymously. The Client generates
+a stable ANON_ORIGIN_ID for each ORIGIN_NAME, to allow the Mediator to count
+token access without learning the ORIGIN_NAME.
+
+CLIENT_GROUP_ID:
+: An identifier chosen by the Mediator and sent to the Issuer, which represents
+a random cohort of Clients. Each Client is assigned a new CLIENT_GROUP_ID for each
+Issuer it uses every time a new policy window begins.
+
+GROUP_ORIGIN_ID:
+: An identifier chosen by the Issuer and sent to the Mediator, which corresponds
+to the tuple of an ORIGIN_NAME and a CLIENT_GROUP_ID. This identifier is used by
+the Mediator to ensure that Clients cannot create a mismatch between an
+ORIGIN_NAME and an ANON_ORIGIN_ID.
 
 # API Endpoints {#setup}
 
 Issuers MUST provide the following information available via the corresponding
 API endpoints:
 
-- Issuer name public key (ISSUER_NAME_KEY): /.well-known/issuer-name-key
-- Access token policy window (ISSUER_POLICY_WINDOW): /.well-known/access-token-window
+- ISSUER_NAME_KEY: /.well-known/issuer-name-key
+- ISSUER_POLICY_WINDOW: /.well-known/issuer-policy
 
 The content of issuer name public key is a `KeyConfig` as defined in {{!OHTTP=I-D.thomson-http-oblivious}}
 to use when encrypting the ORIGIN_NAME in issuance requests. The response uses media type
 "application/ohttp-keys".
 
-The access token policy window is a resource of media type "application/json", with the
-following structure:
+The policy window (ISSUER_POLICY_WINDOW) is store as a resource of media type "application/json",
```

Comment:
```suggestion
The policy window (ISSUER_POLICY_WINDOW) is a resource of media type "application/json",
```

This can be stored at the Mediator and Issuer in any form; it's just communicated as app/json